backend/shork/src/test/kotlin/instruction/TestDjn.kt

```diff
@@ -0,0 +1,222 @@
+package instruction
+
+import kotlin.test.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import software.shonk.interpreter.internal.InternalShork
+import software.shonk.interpreter.internal.addressing.AddressMode
+import software.shonk.interpreter.internal.addressing.Modifier
+import software.shonk.interpreter.internal.instruction.Dat
+import software.shonk.interpreter.internal.instruction.Djn
+import software.shonk.interpreter.internal.program.Program
+import software.shonk.interpreter.internal.settings.InternalSettings
+
+internal class TestDjn {
+
+    private val dat = Dat(0, 0, AddressMode.IMMEDIATE, AddressMode.IMMEDIATE, Modifier.A)
+    private val settings = InternalSettings(8000, 1000, dat, 1000, 100)
```

Comment:
Your tests are failing because you now need to add the MockGameDataCollector in the settings.
For an example, see https://github.com/corewar-teamprojekt/corewar/blob/main/backend/shork/src/test/kotlin/instruction/TestJmp.kt#L19